src/CORS/src/Microsoft.AspNetCore.Cors/Infrastructure/CorsEndpointConventionBuilderExtensions.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Routing;
+using System;
+using System.Linq;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    public static class CorsEndpointConventionBuilderExtensions
+    {
+        public static IEndpointConventionBuilder RequireCors(this IEndpointConventionBuilder builder, string policyName)
```

Comment:
can policy name by null? What does that mean?

Reply:
A null policy name means either the default policy or default policy name configured passed to the middleware constructor is used.